bindings/python/src/lib.rs

```diff
@@ -51,18 +51,18 @@ pub fn call_utils_method(method: String) -> Result<String> {
 pub fn migrate_stronghold_snapshot_v2_to_v3(
     current_path: String,
     current_password: String,
-    salt: String,
+    salt: &str,
     rounds: u32,
     new_path: Option<String>,
     new_password: Option<String>,
 ) -> Result<()> {
     Ok(StrongholdAdapter::migrate_snapshot_v2_to_v3(
         &current_path,
-        &current_password,
-        &salt,
+        current_password.into(),
```

Comment:
Any reason not to take an Into for this function?

Reply:
if you feel strongly about it I can change that too. My reasoning was that `migration` is special in the sense that it's not repeatedly used (right?), hence the convenience aspect is kind of irrelevant. So I wanted to discuss whether it's worth making the function signature more complicated adding another generic.

Reply:
I don't, was just wondering, fair enough